store.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Karma - Store</title>
+</head>
+<body>
+  <!-- I have deliberately completed this WITHOUT any IDs or Classes for Specificity/Selector Practise -->
+  <header>
+    <a href="./index.html"><img src="./img/karma-logo.svg" alt="Karma Logo"></a>
+    <nav>
+      <ul>
+        <li><a href="./index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a href="./store.html">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+      <a href="#"><img src="./level-3/menu-hamburger.svg" alt="Hamburger Menu"></a>
+    </nav>
+  </header>
+  <main>
+   <section>
+    <h1>Order your Karma wifi device today!</h1>
+    <form action="">
+      <div>
+        <label for="first-name">First name *</label>
+        <input 
+          type="text" 
+          id="first-name"
+          value = ""
+          pattern="^(\w\w+)$"
+          title="A valid first name has 2+ characters"
```

Comment:
Really good that you are adding help text but the title attribute has problems of only being visible to mouse users and only after a delayed period of time. It also has issues for screen readers so is generally discouraged. Options here would be to just show it all the time or show it as an error message